tesseract/tesseract_planning/tesseract_process_managers/include/tesseract_process_managers/process_input.h

```diff
@@ -52,29 +52,38 @@ struct ProcessInput
   ProcessInput(tesseract::Tesseract::ConstPtr tesseract,
                const Instruction* instruction,
                const ManipulatorInfo& manip_info,
-               Instruction* seed);
+               Instruction* seed,
+               bool verbose = false);
 
   ProcessInput(tesseract::Tesseract::ConstPtr tesseract,
                const Instruction* instruction,
                const ManipulatorInfo& manip_info,
                const PlannerProfileRemapping& plan_profile_remapping,
                const PlannerProfileRemapping& composite_profile_remapping,
-               Instruction* seed);
+               Instruction* seed,
+               bool verbose = false);
 
   ProcessInput(tesseract::Tesseract::ConstPtr tesseract,
                const Instruction* instruction,
                const PlannerProfileRemapping& plan_profile_remapping,
                const PlannerProfileRemapping& composite_profile_remapping,
-               Instruction* seed);
+               Instruction* seed,
+               bool verbose = false);
 
-  ProcessInput(tesseract::Tesseract::ConstPtr tesseract, const Instruction* instruction, Instruction* seed);
+  ProcessInput(tesseract::Tesseract::ConstPtr tesseract,
+               const Instruction* instruction,
+               Instruction* seed,
+               bool verbose = false);
 
   /** @brief Tesseract associated with current state of the system */
   const tesseract::Tesseract::ConstPtr tesseract;
 
   /** @brief Global Manipulator Information */
   const ManipulatorInfo& manip_info;
 
+  /** @brief Verbose Output */
+  bool verbose{ false };
```

Comment:
I wonder if this should be an int so we can have different verbosity levels.

Reply:
I was thinking the same thing after noticing that in some places we were using the console bridge log level to control output. Should we just use that as a parameter?